websites/C/Crunchyroll/presence.ts

```diff
@@ -180,15 +163,31 @@ presence.on("UpdateData", async () => {
 		if (videoTitle) {
 			presenceData.buttons = [
 				{
-					label: (await strings).watchEpisode,
-					url: document.location.toString(),
+					label: strings.watchEpisode,
+					url: href,
 				},
 				{
-					label: (await strings).viewSeries,
+					label: strings.viewSeries,
 					url: seriesLink,
 				},
 			];
-			presence.setActivity(presenceData, !paused);
 		}
+	} else if (pathname.includes("/series")) {
+		presenceData.details = strings.viewPage;
+		presenceData.state = document.querySelector("h1.title").textContent;
+		presenceData.largeImageKey =
+			document.querySelector<HTMLMetaElement>("[property='og:image']")
+				?.content ?? "lg";
```

Comment:
maybe just set it to an imgur link instead of lg. For future maintainability

Reply:
👍 I'll change that when I can

Reply:
Resolved in https://github.com/PreMiD/Presences/pull/6641/commits/e33d6786dfb5f4f5383997f4349c6afdb3842ccf